README.md

```diff
@@ -94,7 +94,7 @@ ps.  Quoted from [s5cmd feedstock](https://github.com/conda-forge/s5cmd-feedstoc
 You can build `s5cmd` from source if you have [Go](https://golang.org/dl/) 1.17+
 installed.
 
-    go get github.com/peak/s5cmd
+    go install github.com/peak/s5cmd/v2
```

Comment:
I think the path needs to be something like `github.com/peak/s5cmd/v2@master` or `github.com/peak/s5cmd@master` ;
```bash
; go version
go version go1.20.5 darwin/amd64

; go install github.com/peak/s5cmd/v2
go: 'go install' requires a version when current directory is not in a module
        Try 'go install github.com/peak/s5cmd/v2@latest' to install the latest version
```

Omitting version or using `@latest` would try to build latest versioned release AFAIK;
> https://go.dev/ref/mod#version-queries
> The string latest, which selects the highest available release version.

Reply:
PTAL